livedata/src/main/java/io/getstream/chat/android/livedata/entity/MessageEntity.kt

```diff
@@ -83,28 +83,39 @@ data class MessageEntity(@PrimaryKey var id: String, var cid: String, var userId
         }
 
         // add to latest reactions
+        latestReactions = latestReactions.toMutableList()
```

Comment:
Because `latestReaction` is already a MutableList, this line shouldn't be needed

Reply:
calling toMutableList creates a copy of the object. this ensures that we're not modifying the same object in place.